lab_2/main.py

```diff
@@ -33,7 +63,20 @@ def get_language_features(language_profiles: dict) -> list or None:
         and sorts them in alphabetical order
     :param language_profiles: a dictionary of dictionaries - language profiles
     """
-    pass
+    if not isinstance(language_profiles, dict):
+        return None
+
+    language_features = []
+
+    for values in language_profiles.values():
+        for key in values.keys():
+            if key not in language_features:
+                language_features.append(key)
+
+    for feature in language_features:
+        if not isinstance(feature, str):
+            return None
+        return sorted(language_features)
```

Comment:
This line is executed only once (as it is a `return`) - should it be really inside the loop body?